src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -148,7 +149,7 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
 
     @Override
     public Predicate<? super Person> getFilteredPersonListPredicate() {
-        return filteredPersons.getPredicate();
+        return Optional.ofNullable(filteredPersons.getPredicate()).orElse(p -> true);
```

Comment:
Can the readability be improved by replacing p -> true with PREDICATE_SHOW_ALL_PERSONS?